blog/_posts/2020-03-31-extensions.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: more-customizable-tiltfiles-with-extensions
+date: 2020-03-31
+author: victor
+layout: blog
+title: "More customizable Tiltfiles with Extensions"
+subtitle: "Share Tiltfile code snippets with the Tilt community in a streamlined platform. And get a free t-shirt!"
+image: "/assets/images/more-customizable-tiltfiles-with-extensions/michael-dziedzic-XTblNijO9IE-unsplash.jpg"
+image_caption: "Photo by <a href='https://unsplash.com/@lazycreekimages?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText'>Michael Dziedzic</a> on <a href='https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText'>Unsplash</a>"
+tags:
+  - tilt  
+keywords:
+  - extensions
+  - tilt
+  - tiltfile
+---
+
+Since [releasing Tilt](/2018/08/28/how-tilt-updates-kubernetes-in-seconds-not-minutes.html), we’ve seen the Tilt community actively share code snippets of Tiltfile functionality in [Slack #tilt](https://kubernetes.slack.com/messages/CESBL84MV/), including ideas such as forcing resources into different namespaces, injecting sidecars, and even running tests. So we've created the open source Extensions platform to encourage this community collaboration in more structured fashion.
```

Comment:
I feel like this dumps a ton context on the reader. In the first sentence, it jumps around in time, and introduces several features, without a lot of guidance and what to focus on. I have a hard time parsing it, and I'm pretty familiar with Tilt :)

```
Tilt lets you declare your dev environment in Starlark, a subset of Python.

But Starlark is missing a big feature! There's no package manager. Starlark doesn't have built-in ways to share your configuration.

The community noticed! We saw people:
- copy and paste "best practices" for different languages
- struggle to share Tiltfiles across multiple repos
- send each other Tiltfile snippets

Tilt extensions are a first baby step towards a multi-service dev package manager.
```

and maybe at the bottom of the post plug the types of extensions we're hoping to see?

Reply:
I added this to the top section and pretty much removed most of the original paragraph.

> and maybe at the bottom of the post plug the types of extensions we're hoping to see?

Which types of extensions are we hoping to see? I'm not sure "forcing resources into different namespaces, injecting sidecars, and even running tests" are necessarily things we want?